pkg/transfers/inbound/cleanup_test.go

```diff
@@ -43,3 +72,149 @@ func TestCleanupErr(t *testing.T) {
 		t.Errorf("unexpected deleted file: %s", agent.DeletedFile)
 	}
 }
+
+func Test_CleanupEmptyFiles_InboundPath_Success(t *testing.T) {
+	agent := &upload.MockAgent{}
+
+	dir, _ := ioutil.TempDir("", "clenaup-testing")
+	dl := &downloadedFiles{dir: dir}
+	defer dl.deleteFiles()
+
+	// write a test file to attempt deletion
+	path := filepath.Join(dl.dir, agent.InboundPath())
+	if err := os.MkdirAll(path, 0777); err != nil {
+		t.Fatal(err)
+	}
+
+	if err := ioutil.WriteFile(filepath.Join(path, "empty_file.ach"), []byte(""), 0644); err != nil {
+		t.Fatal(err)
+	}
+
+	files, err := ioutil.ReadDir(path)
+	if err != nil {
+		t.Fatal(err)
+	}
+	fileInfo := files[0]
+	mockTimeNow := fileInfo.ModTime().Add(10 * time.Minute)
+	minutesAfterToDelete := 5 * time.Minute
+
+	if err := CleanupEmptyFiles(log.NewNopLogger(), agent, dl, mockTimeNow, minutesAfterToDelete); err == nil {
+		t.Error("expected error")
+	}
+
+	if agent.DeletedFile != "inbound/empty_file.ach" {
+		t.Errorf("unexpected deleted file: %s", agent.DeletedFile)
+	}
+}
+
+func Test_CleanupEmptyFiles_ReturnPath_Success(t *testing.T) {
+	agent := &upload.MockAgent{}
+
+	dir, _ := ioutil.TempDir("", "clenaup-testing")
```

Comment:
I think it'd be better to use `t.TempDir()` here instead because it cleans up after itself. 

ioutil TempDir:
> It is the caller's responsibility to remove the directory when no longer needed.

Compare to testing TempDir:
> The directory is automatically removed by Cleanup when the test and all its subtests complete.

Reply:
I'm not sure on this one. Since the code being tested actually cleans up the files, I'm not sure if having it cleaned up automatically is what we want. Maybe? Need to look into it.

Reply:
It _kind of_ cleans up. It cleans up the files, but leaves the directory. I think the testing package's version of the method makes more sense since it's purpose built for tests.